Controller/SettingsController.php

```diff
@@ -17,22 +15,22 @@ class SettingsController extends Controller {
 
 	public function modifyAction(Request $request) {
 		$em = $this->getDoctrine()->getManager();
-		$repo = $em->getRepository(get_class(new Setting()));
+		$repo = $em->getRepository('Craue\ConfigBundle\Entity\Setting');
 		$allStoredSettings = $repo->findAll();
 
 		$formData = array(
 			'settings' => $allStoredSettings,
 		);
 
-		$form = Kernel::VERSION_ID < 20800
-			? $this->createForm(new LegacyModifySettingsForm(), $formData)
-			: $this->get('form.factory')->createNamed('craue_config_modifySettings', 'Craue\ConfigBundle\Form\ModifySettingsForm', $formData);
+		$form = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
+			? $this->get('form.factory')->createNamed('craue_config_modifySettings', 'Craue\ConfigBundle\Form\ModifySettingsForm', $formData)
```

Comment:
hmm, indeed. ``FormFactory::create`` does not deal with customized block prefixes properly. It always turns the FQCN to a block prefix (similar to the default implementation of the method).

This should be reported to Symfony as a bug in the 2.8 implementation.